spec/controllers/admin/users_controller_spec.rb

```diff
@@ -3,10 +3,35 @@
 describe Admin::UsersController do
 
   describe 'GET index' do
-    subject { get :index }
+    subject(:get_index) { get :index }
 
     authenticated_as(:admin) do
       it { should be_success }
+
+      describe "populating @users" do
+        subject(:users) { get_index; assigns(:users) }
+
+        it { should include(FactoryGirl.create(:user, :member)) }
+        it { should_not include(FactoryGirl.create(:user, :admin)) }
+      end
+    end
+
+    it_behaves_like "action requiring authentication"
+    it_behaves_like "action authorizes roles", [:admin]
+  end
+
+  describe 'GET index_admins' do
+    subject(:get_index) { get :index_admins }
+
+    authenticated_as(:admin) do
+      it { should be_success }
+
+      describe "populating @users" do
```

Comment:
I like this helper:

```ruby
module DescribeHelpers
  def describe_assign(var_name, options={}, &block)
    describe "@#{var_name}" do

      # Mangle the name of subject in such a way that we are very unlikely to change
      # what it is. Object ID returns the unique ID of the current object. In this case the
      # class which defines the current example group.
      mangled_subject = "#{object_id}_subject"
      alias_method(mangled_subject, :subject)

      options[:as] ||= var_name unless instance_methods.include?(var_name.to_sym)
      options[:as] ||= "#{object_id}_imp_subject"

      subject(options[:as]) do
        __send__(mangled_subject)
        assigns[var_name]
      end

      instance_eval &block
    end
  end
end
```

Simplify things a little bit....

```ruby
describe_assign(:users) do
  it { should include blahh }
end
```

If you guys think it is worthwhile, then I think we should add it.